docs/administration/back_office/back_office_elements/reusable_components.md

```diff
@@ -132,51 +130,179 @@ The table component consists of the following blocks:
+- icon column `{ has_icon: true }`
+- checkbox column `{ has_checkbox: true }`
+- action buttons column `{  }` 
+
+with additional parameters available for all of the objects mentioned earlier:
+ 
+    - class (CSS class)
+    - attr (HTML attributes)
+
+See the example:
+
+```html+twig
+{
+    content: 'foo',
+    class: 'bar',
+    attr: {
+        colspan: 2,
+    },
 
-    {% block tbody %}
-        {% for i in 1..10 %}
-            <tr>
-                <td></td>
-                <td>A{{ i }}</td>
-                <td>B{{ i }}</td>
-                <td>C{{ i }}</td>
-            </tr>
-        {% endfor %}
-    {% endblock %}
-{% endembed %}
 ```
 
+- `empty_table_info_text` and `empty_table_action_text` specify texts which are displayed when the table is empty.
+
+
+```html+twig
+{{ form_start(form_remove, {
+    action: path('ibexa.article.remove'),
+    attr: { class: 'ibexa-toggle-btn-state', 'data-toggle-button-id': '#article_remove_remove' }
+}) }}
+{% include '@ibexadesign/ui/component/table/table.html.twig' with {
+    headline: results_headline(pager.getNbResults()),
+    head_cols: [
+        { has_checkbox: true },
+        { has_icon: true },
+        { content: 'article.list.name'|trans|desc('Name') },
+        { content: 'article.list.content_type'|trans|desc('Content Type') },
+        { },
+    ],
+    body_rows,
+    actions: form_widget(form_remove.remove, { attr:
+        {
+            class: 'btn ibexa-btn ibexa-btn--ghost ibexa-btn--small',
+            disabled: true,
+        }}),
+    empty_table_info_text: 'article.list.empty'|trans|desc('You have no articles yet. Your articles will show up here.'),
+    empty_table_action_text: 'article.list.empty_desc'|trans|desc('Articles you create will show up here.'),
+} %}
+{{ form_end(form_remove) }}
+```
+
+Other table component parameters include:
+
+- `class` - (CSS table class)
+- `attr` - (other HTML attributes applied on the HTML table element), for example:
+    - `attr: { 'data-some-data-attribute-you-need': 'foo' }`
+- `table_body_class` and `table_body_attr` are the same as mentioned earlier, but applied on the table element
+- `show_head_cols_if_empty` - (default: `false`), by default, when `body_rows` is empty, the table component does not show the table header, but you may want to have it because for example rows are rendered dynamically with JavaScript on the browser side.
+
+To avoid wrapping headline inside the form, as it's done in the earlier example, you can `embed` table and override the `between_header_and_table` block:
+
+```html+twig
+{% block between_header_and_table %}
+    {{ form_start(form_remove, {
+        action: path('ibexa.article.remove'),
+        attr: { class: 'ibexa-toggle-btn-state', 'data-toggle-button-id': '#article_remove_remove' }
+    }) }}
+{% endblock %}
+```
+
+This method is useful in case of another form inside headline actions or to avoid interferences with the form like button triggering its sumbmission.
```

Comment:
```suggestion
This method is useful in case of another form inside headline actions or to avoid interferences with the form like button triggering its submission.
```